frontend/src/components/eMIB/TeamInformation.jsx

```diff
@@ -49,66 +52,117 @@ class TeamInformation extends Component {
     this.setState({ showPopupBox: false });
   };
 
+  // populates the tree view array with people names
+  populateTreeArray = response => {
+    const responsePrefixEn =
+      response.background.en.background[0].tree_view[1].team_information_tree_child;
+    const responsePrefixFr =
+      response.background.fr.background[0].tree_view[1].team_information_tree_child;
+    // populating names in English
+    for (let i = 0; i < responsePrefixEn.length; i++) {
+      this.setState({ treeViewContent_en: responsePrefixEn[i] });
+    }
+    // populating names in French
+    for (let i = 0; i < responsePrefixFr.length; i++) {
+      this.setState({ treeViewContent_fr: responsePrefixFr[i] });
+    }
+  };
+
   // loads the markdown content (english and french versions)
   componentWillMount = () => {
     this.props.getTestQuestions(TEST_DEFINITION.emib.sampleTest).then(response => {
+      this.populateTreeArray(response);
       // saving the team information markdown content in local states
       this.setState({
         markdown_section1_en: response.background.en.background[0].markdown[3].text,
         markdown_section1_fr: response.background.fr.background[0].markdown[3].text,
         markdown_section2_en: response.background.en.background[0].markdown[4].text,
-        markdown_section2_fr: response.background.fr.background[0].markdown[4].text
+        markdown_section2_fr: response.background.fr.background[0].markdown[4].text,
+        isLoadingComplete: true
       });
     });
   };
 
   render() {
     const { currentLanguage } = this.props;
-    const treeView = [
-      {
-        id: 0,
-        name: LOCALIZE.emibTest.background.teamInformation.dialog.manager,
-        groups: [1, 2, 3, 4, 5, 6],
-        level: 1
-      },
-      {
-        id: 1,
-        name: LOCALIZE.emibTest.background.teamInformation.dialog.analyst1,
-        parent: 0,
-        level: 2
-      },
+    const { treeViewContent_en, treeViewContent_fr } = this.state;
+    let treeView = [];
+    // waiting for tree view content data loading
+    if (this.state.isLoadingComplete) {
+      treeView = [
+        {
+          id: 0,
+          name: `${
+            currentLanguage === LANGUAGES.english
+              ? treeViewContent_en.text
+              : treeViewContent_fr.text
+          }`,
+          groups: [1, 2, 3, 4, 5, 6],
+          level: 1
```

Comment:
As discussed in #422, we will use the hybrid hard-coded/loaded solution for now; a later PR will move to loading entirely from the DB (just noting here for consistency/explanation)

Reply:
I agree - I think this is progress, but there will be more work when we want to add the pizza test.